README.md

```diff
@@ -1026,6 +1026,24 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 6.26. Have a security.txt File [PRODUCTION]
+
+**TL;DR:** Have a text file called ```security.txt``` under ```/.well-known```  directory (/.well-known/security.txt) or in the root directory (/security.txt) of your website or your Web Application in Production. ```security.txt``` file should contain details using which Security Researchers can report Vulnerabilities and also the contact details of the responsible person/group (email id and/or phone numbers) to whom the reports have to be sent. 
```

Comment:
I think we can have the language here as lower case in most cases, for example: `Security Researchers` , `Vulnerabilities`, `Production`

Reply:
Updated